src/components/Table/Table/__stories__/table.stories.js

```diff
@@ -261,9 +265,135 @@ export const Sizes = {
+        loadingStateType: "medium-text"
+      },
+      {
+        id: "emailsSent",
+        title: "Emails sent",
+        width: 150,
+        loadingStateType: "medium-text"
+      }
+    ];
+    const data = [
+      {
+        id: "1",
+        sentOn: "2020-01-01",
+        sentBy: "John Doe",
+        subject: "Lorem ipsum dolor",
+        status: "Sent",
+        emailsSent: 100
+      },
+      {
+        id: "3",
+        sentOn: "2023-03-03",
+        sentBy: "Some Person",
+        subject:
+          "This is the subject This is the subject This is the subject This is the subject This is the subject This is the subject",
+        status: "Sent",
+        emailsSent: 999
+      },
+      {
+        id: "2",
+        sentOn: "2022-02-02",
+        sentBy: "Other Name",
+        subject: "This is the subject",
+        status: "Sent",
+        emailsSent: 99
+      }
+    ];
+    return (
+      <>
+        <Table errorState={<TableErrorState />} emptyState={<TableEmptyState />} columns={columns}>
+          <TableHeader>
+            {columns.map((headerCell, index) => (
+              <TableHeaderCell key={index} title={headerCell.title} />
+            ))}
+          </TableHeader>
+          <TableBody>
+            {data.map(rowItem => (
+              <TableRow key={rowItem.id}>
+                <TableCell>{rowItem.sentOn}</TableCell>
+                <TableCell>{rowItem.subject}</TableCell>
+                <TableCell>
+                  <TableAvatar text={rowItem.sentBy} />
+                </TableCell>
+                <TableCell>
+                  <Label text={rowItem.status} color="positive" isAnimationDisabled />
+                </TableCell>
+                <TableCell>{rowItem.emailsSent}</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+        <Table errorState={<TableErrorState />} emptyState={<TableEmptyState />} columns={columns} withoutBorder>
+          <TableHeader>
+            {columns.map((headerCell, index) => (
+              <TableHeaderCell key={index} title={headerCell.title} />
+            ))}
+          </TableHeader>
+          <TableBody>
+            {data.map(rowItem => (
+              <TableRow key={rowItem.id}>
+                <TableCell>{rowItem.sentOn}</TableCell>
+                <TableCell>{rowItem.subject}</TableCell>
+                <TableCell>
+                  <TableAvatar text={rowItem.sentBy} />
+                </TableCell>
+                <TableCell>
+                  <Label text={rowItem.status} color="positive" isAnimationDisabled />
+                </TableCell>
+                <TableCell>{rowItem.emailsSent}</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      </>
+    );
+  },
+  decorators: [
+    Story => (
+      <Flex direction={Flex.directions.COLUMN} gap={40}>
+        <Story />
+      </Flex>
     )
   ],
   name: "Sizes"
```

Comment:
I think the name here needs to be "Borders"

Reply:
Oops
Nice catch, thanks!